Source/WebInspectorUI/UserInterface/Controllers/CSSManager.js

```diff
@@ -306,6 +307,19 @@ WI.CSSManager = class CSSManager extends WI.Object
         return this._defaultAppearance;
     }
 
+    get defaultUserPreferences()
+    {
+        return this._defaultUserPreferences;
+    }
+
+    set defaultUserPreferences(userPreferences)
+    {
+        for (let userPreference of userPreferences)
+            this.defaultUserPreferences.set(userPreference.name, userPreference.value)
```

Comment:
Should we also do a `this._defaultUserPreferences.clear()` at the start of this function?

Style: please use `_defaultUserPreferences` since we're inside the class that declares the underlying member variable

Reply:
Removed the setter per other comment. 

I'll not clear the `this._defaultUserPreferences` on this call yet. I have a follow-up patch that's implementing a backwards compatible appearance override. The handler for the old `defaultAppearanceDidChange` event will add its value to `this._defaultUserPreferences`. A subsequent clear would drop it because the `defaultUserPreferencesDidChange` won't have the value for the legacy default appearance.

Reply:
we probably should have at least _some_ way of clearing values from `_defaultUserPreferences` 🤔